en/frames_multicopter/omnicopter.md

```diff
@@ -0,0 +1,107 @@
+  - 3D Propellers: 2x [HQProp 3D 5X3.5X3 3-Blade Propeller (Set of 4)](https://www.getfpv.com/hqprop-3d-5x3-5x3-3-blade-propeller-set-of-4.html) or 2x [Gemfan 513D 3-Blade 3D Propeller (Set of 4)](https://www.getfpv.com/gemfan-513d-durable-3-blade-propeller-set-of-4.html)
+  - Battery: we used a 6S 3300mAh LiPo. Make sure to check the dimensions so it fits the frame.
+  - Battery strap
+- Frame:
+  - Carbon square tube R 8mm X 7mm X 1000mm, e.g. [here](https://shop.swiss-composite.ch/pi/Halbfabrikate/Rohre/Vierkant-Rohre/CFK-Vierkantrohr-8x8-7x7mm.html)
+  - Carbon Rods R 3mm X 2mm X 1000mm, e.g. [here](https://shop.swiss-composite.ch/pi/Halbfabrikate/Rohre/CFK-Rohre-pultrudiert-pullwinding/Carbon-Microtubes-100cm-x-20-3mm.html)
+- TODO: 3D parts + screws + standoffs + rod lengths
+
+![Parts List](../../assets/airframes/multicopter/omnicopter/parts_list.jpg)
+
+## Assembly
+
+### Frame
+- Print the 3D parts
+- Cut the rods
+- Test that it all works by connecting the frame pieces together:
+![Frame](../../assets/airframes/multicopter/omnicopter/frame_only.jpg)
+- Place the motors as far out as possible, without the propellers touching the rods.
+
+### Electronics
+Solder the peripherals to the flight controller. We used the following assignments:
+- ESCs: the 2 ESCs can be connected directly to the two connectors of the KakuteH7. To avoid conflicts we removed the power pin (right-most pin) from one of the connectors.
+- Telemetry to UART1
+- GPS to UART4
+- RC to UART6
+![FC closeup](../../assets/airframes/multicopter/omnicopter/fc_closeup.jpg)
+
+Remarks:
+- Make sure the magnetometer is placed away from power. We ended up placing it to the bottom of the center-piece with a 4cm padding styrofoam.
+- Put some tape on the barometer (without taping the opening!) to avoid any influence from light.
+- We did not glue the frame. It is certainly advisible to do so after initial test flights, but it might work without.
+
+
+## Software Configuration
+
+### ESC
+First, configure the ESCs to 3D mode (bidirectional).
+We had issues with the stock ESC settings in 3D mode: when trying to switch direction, a motor sometimes did not start anymore until the ESC was rebooted.
+So we had to change ESC settings.
+To do so, you can use Betaflight on the flight controller and then use pass-through mode and the BL Heli suite (make sure there is an airframe with 8 motors configured in Betaflight).
+These are the settings:
+![ESC settings](../../assets/airframes/multicopter/omnicopter/esc_settings.png)
+In particular:
+  - set the Motor Direction to **Bidirectional Soft**
+  - increase the Rampup Power to **100%** (this is conservative and might reduce efficiency)
+
+:::note
+Make sure the motors do not overheat with the changed settings.
+:::
+
+### PX4
+- Use an arming switch, do not use stick arming
+- Select DShot as output protocol
+- Configure the motors according to this:
+![Motor Config](../../assets/airframes/multicopter/omnicopter/motors_configuration.png)
```

Comment:
How about attaching a .param file with all the CA_* params defined?

Reply:
Right, will do that